### DISCUSSION TITLE
Module not found: Error: Can't resolve './zlib_bindings'

### DISCUSSION THREAD

**Discussion by azhar-neolysi** [npm/feedback#888] [Registry]
./node_modules/pngjs/lib/sync-inflate.js:4:11-26 - Error: Module not found: Error: Can't resolve 'zlib' in 'D:\Angular\Ionic\printer4\node_modules\pngjs\lib'
[ng]
[ng] BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
[ng] This is no longer the case. Verify if you need this module and configure a polyfill for it.
[ng]
[ng] If you want to include a polyfill, you need to:
[ng]    - add a fallback 'resolve.fallback: { "zlib": require.resolve("browserify-zlib") }'
[ng]    - install 'browserify-zlib'
[ng] If you don't want to include a polyfill, you can use an empty module like this:
[ng]    resolve.fallback: { "zlib": false }

**after intall npm i zlib**  

[ng] ./node_modules/zlib/lib/zlib.js:1:0-43 - Error: Module not found: Error: Can't resolve './zlib_bindings' in 'D:\Angular\Ionic\printer4\node_modules\zlib\lib'

**any solutions**

**Comment by EffiEmmanuel** [comment]
If you are using webpack, add this in the resolve block of your webpack.config.js file, in your react-scripts folder in your node_modules:

```
resolve: {
        fallback: { 
            "zlib": require.resolve('browserify-zlib')
        },
}
```
